index.js

```diff
@@ -63,8 +63,24 @@ class Dynamodb extends Datastore {
         });
     }
 
+    /**
+     * Save a item in the specified DynamoDB table
+     * @param  {Object}   config             Configuration object
+     * @param  {String}   config.table       Table name
+     * @param  {Object}   config.params      Record data
+     * @param  {String}   config.params.id   Unique id. Typically the desired primary key
+     * @param  {Object}   config.params.data The data to save
+     * @param  {Function} callback           fn(err)
```

Comment:
fn(err,data) to be consistent with imdb: 
https://github.com/screwdriver-cd/screwdriver-datastore-imdb/blob/master/index.js#L67